blueprints/module-unification-app/files/src/ui/routes/index/template.hbs

```diff
@@ -0,0 +1,3 @@
+<code>src/ui/routes/index/template.hbs</code>
+
+{{my-component}}
```

Comment:
Should invoke the `{{welcome-page}}` addon...

Reply:
To mirror the `app` blueprint, I'm going to zap the `index` route and add `{{welcome-page}}` to `application.hbs`